.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,71 @@
+name: "🐛 Bug report"
+description: Report for reproducible bug of cli-kintone
+labels: ["bug"]
+title: "[Bug] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting a bug! Please fill out the followings.
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: Please describe what the bug is.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Target Version
+      placeholder: e.g. vX.Y.Z
+      description: Which version of cli-kintone do you want to report a bug for? 
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: |
+        Please provide [minimum reproduction](https://stackoverflow.com/help/minimal-reproducible-example) steps of the bug.
+        If you have a reproduction repository, [GitHub Gist](https://gist.github.com/) snippet, or some external pages, please share the links.
+      placeholder: |
+        1. Prepare a Kintone App with the following fields...
+        2. Run cli-kintone with the following options...
```

Comment:
Please add a warning like "DO NOT SUBMIT CONFIDENTIAL INFORMATION SUCH AS YOUR SUB-DOMAIN AND PASSWORD"